### PR TITLE
feat(route): 支持长沙停水通知

### PR DESCRIPTION
**Pull request by shansing**
## 完整路由地址 / Example for the proposed route(s)

```routes
/tingshuitz/changsha/78
/tingshuitz/changsha/157
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note

相应网站可能仅限中国大陆境内访问，GitHub actions 自动测试请求可能超时。

The website seems to have blocked access outside China, so that requests of auto test may time out.

P.S. The former pull request was automatically closed because I renamed the branch of my fork, so I'm opening this one.

### CODE REVIEW

File: lib/v2/tingshuitz/maintainer.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
+    '/changsha': ['shansing'],
```
Comment (TonyRL) [inline lib/v2/tingshuitz/maintainer.js:2]:
```suggestion
    '/changsha/:channelId?': ['shansing'],
```

File: docs/forecast.md [hunk 2]
```diff
@@ -149,6 +149,19 @@ pageClass: routes
 
 </Route>
 
+### 长沙市
+
+<Route author="shansing" example="/tingshuitz/changsha/78" path="/tingshuitz/changsha/:channelId">
```
Comment (TonyRL) [inline docs/forecast.md:154]:
```suggestion
<Route author="shansing" example="/tingshuitz/changsha/78" path="/tingshuitz/changsha/:channelId?">
```
path here should be the same as route defined in `route.js` and `maintainer.js`

File: lib/v2/tingshuitz/changsha.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            return {
+                title: item.text().trim(),
+                link: baseUrl + item.attr('href').trim(),
+            };
+        })
+        .get();
+
+    const items = await Promise.all(
+        list.map(async (item) => {
+            const postPage = await got(item.link);
```
Comment (TonyRL) [inline lib/v2/tingshuitz/changsha.js:24]: Any reasons why cache is not used?

Comment (shansing) [reply lib/v2/tingshuitz/changsha.js:24]: Well, it's notice of water supply being cut off. The posts are often updated about the progress, like the new estimated recovery time or recovery announcement. So I think it's better to leave the RSS client to decide the fetch interval.

**Pull request merged**